== Bootstrap started Tue, Sep  9, 2025  9:50:40 AM ==

==================== Running terraform apply =====================
[0m[1mInitializing the backend...[0m
[0m[1mUpgrading modules...[0m
- bastion in modules\BASTION
- dns in modules\DNS
- eks in modules\EKS
- rds in modules\RDS
- redis in modules\REDIS
- vpc in modules\VPC
[0m[1mInitializing provider plugins...[0m
- Finding hashicorp/kubernetes versions matching "~> 2.23"...
- Finding hashicorp/aws versions matching "~> 5.0"...
- Using previously-installed hashicorp/kubernetes v2.38.0
- Using previously-installed hashicorp/aws v5.100.0

[0m[1m[32mTerraform has been successfully initialized![0m[32m[0m
[0m[32m
You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.[0m
[31m╷[0m[0m
[31m│[0m [0m[1m[31mError: [0m[0m[1mUnsupported attribute[0m
[31m│[0m [0m
[31m│[0m [0m[0m  on main-bastion-setting.tf line 56, in resource "aws_eks_access_entry" "bastion":
[31m│[0m [0m  56:   cluster_name      = module.eks[4m.cluster_name[0m[0m
[31m│[0m [0m    [90m├────────────────[0m
[31m│[0m [0m[0m    [90m│[0m [1mmodule.eks[0m is a object
[31m│[0m [0m[0m
[31m│[0m [0mThis object does not have an attribute named "cluster_name".
[31m╵[0m[0m
[31m╷[0m[0m
[31m│[0m [0m[1m[31mError: [0m[0m[1mReference to undeclared resource[0m
[31m│[0m [0m
[31m│[0m [0m[0m  on main-root.tf line 143, in resource "aws_instance" "bastion":
[31m│[0m [0m 143:   ami                         = [4mdata.aws_ami.bastion[0m.id[0m
[31m│[0m [0m
[31m│[0m [0mA data resource "aws_ami" "bastion" has not been declared in the root
[31m│[0m [0mmodule.
[31m╵[0m[0m
[31m╷[0m[0m
[31m│[0m [0m[1m[31mError: [0m[0m[1mUnsupported attribute[0m
[31m│[0m [0m
[31m│[0m [0m[0m  on outputs.tf line 15, in output "cluster_name":
[31m│[0m [0m  15:   value       = module.eks[4m.cluster_name[0m[0m
[31m│[0m [0m    [90m├────────────────[0m
[31m│[0m [0m[0m    [90m│[0m [1mmodule.eks[0m is a object
[31m│[0m [0m[0m
[31m│[0m [0mThis object does not have an attribute named "cluster_name".
[31m╵[0m[0m
[31m╷[0m[0m
[31m│[0m [0m[1m[31mError: [0m[0m[1mUnsupported attribute[0m
[31m│[0m [0m
[31m│[0m [0m[0m  on outputs.tf line 20, in output "shopping_mall_service_account":
[31m│[0m [0m  20:   value       = module.eks[4m.shopping_mall_service_account[0m[0m
[31m│[0m [0m    [90m├────────────────[0m
[31m│[0m [0m[0m    [90m│[0m [1mmodule.eks[0m is a object
[31m│[0m [0m[0m
[31m│[0m [0mThis object does not have an attribute named
[31m│[0m [0m"shopping_mall_service_account".
[31m╵[0m[0m
[31m╷[0m[0m
[31m│[0m [0m[1m[31mError: [0m[0m[1mUnsupported attribute[0m
[31m│[0m [0m
[31m│[0m [0m[0m  on outputs.tf line 30, in output "pod_identity_association_id":
[31m│[0m [0m  30:   value       = module.eks[4m.pod_identity_association_id[0m[0m
[31m│[0m [0m    [90m├────────────────[0m
[31m│[0m [0m[0m    [90m│[0m [1mmodule.eks[0m is a object
[31m│[0m [0m[0m
[31m│[0m [0mThis object does not have an attribute named "pod_identity_association_id".
[31m╵[0m[0m
[31m╷[0m[0m
[31m│[0m [0m[1m[31mError: [0m[0m[1mUnsupported attribute[0m
[31m│[0m [0m
[31m│[0m [0m[0m  on outputs.tf line 47, in output "db_endpoint":
[31m│[0m [0m  47:   value       = module.rds[4m.db_endpoint[0m[0m
[31m│[0m [0m    [90m├────────────────[0m
[31m│[0m [0m[0m    [90m│[0m [1mmodule.rds[0m is a object
[31m│[0m [0m[0m
[31m│[0m [0mThis object does not have an attribute named "db_endpoint".
[31m╵[0m[0m
[31m╷[0m[0m
[31m│[0m [0m[1m[31mError: [0m[0m[1mexpected type to be one of ["EC2" "EC2_LINUX" "EC2_WINDOWS" "FARGATE_LINUX" "HYBRID_LINUX" "STANDARD"], got IAM[0m
[31m│[0m [0m
[31m│[0m [0m[0m  with module.bastion.aws_eks_access_entry.bastion,
[31m│[0m [0m  on modules\BASTION\main.tf line 171, in resource "aws_eks_access_entry" "bastion":
[31m│[0m [0m 171:   type             = [4m"IAM"[0m[0m
[31m│[0m [0m
[31m╵[0m[0m

❌ ERROR: line 147 while running: terraform -chdir="$TF_DIR" apply -auto-approve (exit=1)
👉 Full log: ./logs/bootstrap_20250909_095040.log
Dropping you into interactive shell (type: exit to quit)
